portfolio/src/main/webapp/index.html

```diff
@@ -1,74 +1,95 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    s<meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-        <div id = "nav-bar">
-            <button >Home</button>
-            <div id="scroll-buttons">
-            <button id="about-button" onclick="moveToAbout()">About</button>
-            <button id="experience-button" onclick="moveToExperience()">Experience</button>
-            <button id="education-button" onclick="moveToEducation()">Education</button>
-            <button >Projects</button>
-            <button id="interests-button" onclick="moveToInterest()">Interests</button>
-            </div>
+
+<head>
+  <meta charset="UTF-8">
+  <title>My Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <script src="script.js"></script>
+</head>
+
+<body>
+  <div id="content">
+    <div id="nav-bar">
+      <button>Home</button>
+      <div id="scroll-buttons">
+        <button id="about-button" onclick="moveToAbout()">About</button>
+        <button id="experience-button" onclick="moveToExperience()">Experience</button>
+        <button id="education-button" onclick="moveToEducation()">Education</button>
+        <button>Projects</button>
+        <button id="interests-button" onclick="moveToInterest()">Interests</button>
+      </div>
+    </div>
+    <div class="flex-hor">
+      <div>
+        <h1 style="margin-top:50%;"> HI ! I am </h1>
+        <h1 class="prim_color">>Nourish Cherish</h1>
```

Comment:
You have an extraneous `>` here:
```suggestion
        <h1 class="prim_color">Nourish Cherish</h1>
```

Surprising the HTML validator did not catch this.

Reply:
I guess i found a bug in the html validator. haha